src/libs/actions/Policy.js

```diff
@@ -786,7 +801,11 @@ function createWorkspace(ownerEmail = '', makeMeAdmin = false, policyName = '')
         {
             onyxMethod: CONST.ONYX.METHOD.MERGE,
             key: `${ONYXKEYS.COLLECTION.REPORT}${announceChatReportID}`,
-            value: {pendingAction: null},
+            value: {
```

Comment:
Don't we still have to set `pendingAction` to null in all 3 of these?

Reply:
@eVoloshchak I just added `pendingAction` to null, thanks for pointing out that